packages/components/src/Base/FC__E.re

```diff
@@ -105,6 +118,31 @@ module L = {
   let update = Rationale.RList.update;
   let iter = List.iter;
   let findIndex = Rationale.RList.findIndex;
+  module React = {
+    let map = (map: 'a => React.element, list: list('a)): React.element => {
```

Comment:
This function seems suspicious to me. Can you not do it with something like ``List.map`` or one of the Rationale List methods?